package.json

```diff
@@ -57,5 +57,11 @@
     "@typescript-eslint/parser": "^5.30.0",
     "eslint": "^8.13.0",
     "typescript": "^4.7.2"
+  },
+  "prettier": {
```

Comment:
Added a prettier config to make sure my prettier defaults didn't screw up the PR